index.js

```diff
@@ -1,3 +1,13 @@
+class TonicRaw {
+  constructor (rawText) {
+    this.isTonicRaw = true
+    this.rawText = rawText
+  }
+
+  valueOf () { return this.rawText }
+  toString () { return this.rawText }
+}
+
```

Comment:
what do you think about instead of a this being a class, we just return an object from `Tonic.raw()`? Seems equally as testable but less complex.

```
raw (rawTonicText) {
  return { rawTonicText }
}
```

Reply:
I wanted to add `valueOf()` and `toString()` so that if anyone calls `comp.render()` expecting a string it will auto cast to a string.

Instead of casting to `[object Object]` because it's an object with a `rawText` field. 

This improves anyone doing anything funky like

```
class MyComp extends Tonic {
  constructor () {
    this.inner = new OtherComp()
  }

  render () {
    return this.html`<header>...</header>${this.inner.render()}`
  }
}
```

Or in case anyone is trying to implement server side rendering of tonic components by calling `render()` etc ;

Reply:
Otherwise the update for `this.html` to return a `Tonic.raw()` would be a breaking change.